lib/active_model_serializers/test.rb

```diff
@@ -0,0 +1,5 @@
+module ActiveModelSerializers
+  module Test
+    autoload :Schema, 'active_model_serializers/test/schema'
```

Comment:
- [ ] elsewhere we're using `  extend ActiveSupport::Autoload` in place of the ruby autoload.  we can do this in a followup PR